.markdownlint-cli2.yaml

```
@@ -0,0 +1,23 @@
+config:
+  # MD013/line-length - Allow longer lines for documentation
+  MD013:
+    line_length: 120
+    code_blocks: false
+    tables: false
+    headings: false
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)